configure.ac

```diff
@@ -19,6 +19,16 @@ if test "$PKG_CONFIG" = ""; then
   AC_MSG_ERROR([pkg-config not found])
 fi
 
+if test -n "$PKG_CONFIG_PATH"; then
```

Comment:
I think we could add a note here.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/25687#issuecomment-1196481084).